packages/rancher-logging/generated-changes/overlay/templates/loggings/rke2/logging-rke2-journald.yaml

```diff
@@ -22,8 +22,8 @@ spec:
       podSecurityPolicyCreate: true
       roleBasedAccessControlCreate: true
     {{- end }}
-  {{- if .Values.additionalLoggingSources.rke2.fluentbit.mountPath }}
-    mountPath: {{ .Values.additionalLoggingSources.rke2.fluentbit.mountPath }}
+  {{- if .Values.fluentbit.mountPath }}
+    mountPath: {{ .Values.fluentbit.mountPath }}
```

Comment:
Why don't we support both the cases:
1. Have support for separate mounthPaths in Values.yaml
2. If user does not set a separate mountPath, use the ".Values.fluentbit.mountPath"

Reply:
I weighed this idea as well, but did not want to expose the option if it would never realistically get used. My concern was that users may be confused by the option existing in `values.yaml`. Perhaps, we can support both cases and add a comment clarifying that `fluentbit.mountPath` will affect every Logging CR unless a value is detected for `additionalLoggingSources.${SOURCE}.fluentbit.mountPath`.

Reply:
yes comment, or readme can carry the chart values and explanations